package.json

```
@@ -45,8 +40,14 @@
   ],
   "license": "MIT",
   "main": "index.js",
+  "dependencies": {},
+  "devDependencies": {
+    "eslint": "^8.0.0",
+    "eslint-plugin-react": "^7.26.1",
+    "tape": "^5.0.1"
+  },
   "peerDependencies": {
-    "eslint": "^7.12.1",
+    "eslint": "^7.12.1 || ^8.0.0",
```

I would propose changing this to something that works better with npm 7 going forward:

```suggestion
    "eslint": ">=7.12.1",
```

---

I think that the odds of us working with ESLint 9.0.0 without changes are quite low right? Wouldn't it be better to make sure that Npm keeps version 8?

Or do you mean something else with "works better with npm 7 going forward"?